worlds/ahit/test/TestActs.py

```diff
@@ -0,0 +1,35 @@
+from worlds.ahit.Regions import act_chapters
```

Comment:
bad imports

Reply:
Trying to import these in any other way gives me an error.

Reply:
Never mind, just had to change the working directory